data/tutorials/gs_00_up_and_running.md

```diff
@@ -27,90 +27,119 @@ for the instruction.
 
 ### Installing opam
 
 Alternatively, you can use the alpha-version platform installer to install opam and the development tools, [see below](#up-and-running-with-the-platform-installer).
 
-[opam](https://opam.ocaml.org/) is the package manager of OCaml. It introduces the concept of "switches," consisting of a compiler together with a set of packages (libraries and other files). Switches are used to have independent sets of dependencies in different projects.
+[opam](https://opam.ocaml.org/) introduces the concept of "switches," consisting of a compiler together with a set of packages (libraries and other files). Switches are used to have independent sets of dependencies in different projects.
 
 After having installed opam, you will need to initialise it, [see below](#initialize-opam).
 
-To install opam, you can [use your system package manager](https://opam.ocaml.org/doc/Install.html#Using-your-distribution-39-s-package-system), or download the [binary distribution](https://opam.ocaml.org/doc/Install.html#Binary-distribution). The details are available in the above links, but for convenience, we copy a few of them here:
+To install opam, you can [use your system package manager](https://opam.ocaml.org/doc/Install.html#Using-your-distribution-39-s-package-system) or download the [binary distribution](https://opam.ocaml.org/doc/Install.html#Binary-distribution). The details are available in the above links, but for convenience, we copy a few of them here: 
 
 **For macOS**
 
 ```shell
 # With Homebrew:
-brew install opam
+$ brew install opam
 
 # With MacPort:
-port install opam
+$ port install opam
 ```
 
+`# Homebrew` indicates the command if you're using [Homebrew](https://brew.sh/) as the installer, and `# MacPorts` is if you're using [MacPorts](https://www.macports.org/) to install.
+
 **For Linux**
 
+The preferred way to install opam is with your system's package manager on
+Linux (e.g., `apt-get install opam` or similar). [Details of all installation
+methods.](https://opam.ocaml.org/doc/Install.html)
+
 ```shell
 # Ubuntu
-add-apt-repository ppa:avsm/ppa
-apt update
-apt install opam
+$ add-apt-repository ppa:avsm/ppa
+$ apt update
+$ apt install opam
 
 # Archlinux
-pacman -S opam
+$ pacman -S opam
 
 # Debian (stable, testing and unstable)
-apt-get install opam
+$ apt-get install opam
 ```
 
-**Binary distribution**
+**Binary Distribution**
 
 ```shell
-bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)"
+$ bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)"
 ```
 
-### Initialising `opam`
+### Initialising opam
+
+It's essential to both initialise opam and install the OCaml compiler because OCaml is a compiled language, so please don't skip this section. 
```

Comment:
I put this in there because it didn't work for me without this step. 
I can see how the sentence construction is confusing though.

Reply:
Yes, it is indeed necessary to initialize opam! However, I think it is confused to say that it is because OCaml is a compiled language.

How about:
```markdown
It is essential to initialize `opam` (which creates a new switch, installing OCaml if needed) to be able to use it, so do not skip this section!
```
although I find it strange to ask not to skip the section, as I think no section should be skipped!

Reply:
Agreed! I can remove that if you'd like.

Reply:
Yesterday I reworded it like this: It's essential to both initialise `opam` and, because OCaml is a compiled language, install the OCaml compiler.  (today I removed the "skip this section" part)

Does this work for you?

Reply:
There are two things that bother me in your formulation!
- First, the "both" and "and" words seem to suggest that there are two things to do: initialize `opam` and install the OCaml compiler. In fact, there is only one thing to do: initialize `opam`, which will, _if needed_, install an OCaml compiler. But it will use the already installed compiler if it finds it.
- Also, the "because OCaml is a compiled language" seems off here. I don't really see the causal link here!

So I would remove the "because OCaml is a compiled language", and make it clear that there is only one thing to do: initiliaze opam, and that as a consequence of the initialization, it installs the OCaml compiler.

An example of a sentence satisfying (I think!) this specification is:
> It is essential to initialize `opam` (which will install OCaml if needed) to be able to use it.

Reply:
Ah! I got it! It's the same thing! Okay, thanks!